lib/eph/index.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+export {
+  PartitionFiler, EventProcessorHost, OnEphClose, OnEphMessage, OnEphOpen,
```

Comment:
>PartitionFiler [](start = 2, length = 14)

this typo is spread everywhere (thanks to intellisense ;)) recommend searching across the whole library when fixing